src/release/repos.rs

```diff
@@ -287,11 +287,11 @@ X-Repolib-Default-Mirror: http://us.archive.ubuntu.com/ubuntu/
 }
 
 pub fn new_sources_file() -> String {
-    format!(
-        r#"## This file is deprecated in Pop!_OS.
-## See `man deb822` and /etc/apt/sources.list.d/system.sources.
-"#
-    )
+    r###"
+    ## This file is deprecated in Pop!_OS. 
+    ## See `man deb822` and /etc/apt/sources.list.d/system.sources.
+    "###
+    .to_string()
```

Comment:
Trying this on playground, the leading indent shows up in the output.

Reply:
rats. that's embarrassing. try now!